consensus/polybft/stake_manager.go

```diff
@@ -113,19 +116,44 @@ func (s *stakeManager) PostBlock(req *PostBlockRequest) error {
 
 	stakeMap := fullValidatorSet.Validators
 
+	updatedValidatorsBalance := make(map[types.Address]struct{}, 0)
```

Comment:
Rename to `updatedValidatorsStake`?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1464/commits/f550b29ba8f674f866be7aaf24172deff1146bee